bip-0174.mediawiki

```diff
@@ -428,6 +428,72 @@ The currently defined per-input types are defined as follows:
 | 2
 | [[bip-psb2.mediawiki|psbt2]]
 |-
+| Taproot Key Spend Signature
+| <tt>PSBT_IN_TAP_KEY_SIG = 0x13</tt>
+| None
+| No key data 
+| <tt><signature></tt>
+| The 64 or 65 byte Schnorr signature for key path spending a Taproot output.
+|
+|
+| 0, 2
+| bip-taproot-psbt
+|-
+| Taproot Script Spend Signature
+| <tt>PSBT_IN_TAP_SCRIPT_SIG = 0x14</tt>
+| <tt><xonlypubkey> <leafhash></tt>
+| The 32 byte X-only public key concatenated with the 32 byte hash of the leaf it is part of.
```

Comment:
the internal key or the output key?

Reply:
Neither. This field is for any key that appears in a leaf script in the taproot tree.